src/ast.rs

```diff
@@ -389,7 +387,7 @@ impl<'prev, 'subs> ArgScopeStackExt<'prev, 'subs> for Option<ArgScopeStack<'prev
         Some(ArgScopeStack {
             prev: self.as_ref(),
             in_arg: None,
-            item: item,
+            item,
```

Comment:
I don't like `redundant_field_names`

I think it would be good if you disabled that warning and didn't apply those lints.

Reply:
Updated 👍  Hope I got them all.